magento/magento2
# make magento framework LoggerProxy comaptible with psr/log

### Description (*)
Issue comes if some of the third-party modules have requirements limited only to version `^1.0` of the `psr/log` module. Correct way is if magento not support it, to define it in the proper way in the composer.json as requirement, not leave this option to the installer.

Most of the time issue will comes if you are updating magento from previous version (for example 2.4.7-beta2) where changes from commit https://github.com/magento/magento2/commit/28d2a5bf4bd3588dfe6f0d3d0fc2036c6fcc9ef7 are not applied.

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#36083
2. Fixes magento/magento2#38583

### Manual testing scenarios (*)


1. Install magento 2.4.7-beta2 or earlier version.
3. Add `"psr/log": "^1.0"` in the project composer.json file.
4. Update to version 2.4.7-beta3
5. Open magento in the browser. Probably needs to see something which need to be logged. (debug.log is good to be enabled).

### Questions or comments


### Contribution checklist (*)
